string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,11 @@
+/* exported isVowel */
+
+function isVowel(char) {
+  if (char === 'a' || char === 'e' || char === 'i' || char === 'o' || char === 'u') {
+    return true;
+  } else if (char === 'A' || char === 'E' || char === 'I' || char === 'O' || char === 'U') {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
I think you might be able to simplify this by converting the character to either uppercase or lowercase before checking its value.